hazelcast-integration/openshift/hazelcast-cluster/hazelcast-enterprise-openshift-centos/hazelcast-template.json

```diff
@@ -95,15 +79,20 @@
                     ]
                   },
                   "initialDelaySeconds": 0,
-                  "periodSeconds" : 1
+                  "periodSeconds" : 1,
+                  "failureThreshold": 10, 
+                  "timeoutSeconds": 10
                 },
                 "livenessProbe": {
                   "exec": {
                     "command": [
                       "./liveness.sh"
                     ]
                   },
-                  "initialDelaySeconds": 0
```

Comment:
what about using liveness endpoint introduced in 3.10 instead of liveness.sh ?

Reply:
Good point. I've changed the templates.